infra/envs/dev/main.tf

```diff
@@ -92,9 +92,14 @@ module "browser_backend" {
 module "cicd" {
   source = "../../modules/backend/cicd"
 
+  account_id          = data.aws_caller_identity.current.account_id
+  api_gateway_staging = var.api_gateway_staging
+  api_gateway_dev     = var.api_gateway_dev
```

Comment:
Shouldn't these variables be parameterized by environment?

Reply:
e.g. supplying `api_gateway_id` and `deployment_stage` to `github_actions_unittests.tf ` should be sufficient

Reply:
no, these are the api-gateway ids like for dev we have `u5j1w...`. If we setup chalice with terraform then we could use the API-gateway resource in the future.

Reply:
Hm I am referring to the `api-gateway-id` as well. `github_actions_unittests.tf` is a module that is created per env (`dev`, `staging`), correct? `dev/main.tf` would only need to supply the `u5j1w...` ID right?

Reply:
Ah, I see. The module's only included in the dev env. Since this doesn't follow the convention of the rest of our modules, could you add a comment to `github_actions_unittests.tf`?